classes/class-sensei-media-attachments.php

```diff
@@ -113,7 +113,7 @@ public function media_meta_box() {
 		$html .= '</tbody>' . "\n";
 		$html .= '</table>' . "\n";
 
-		echo $html;
+		echo $html; // WPCS: XSS ok.
```

Comment:
Should we use `wp_kses` here for extra safety?

Reply:
Same as above.